x-pack/plugins/spaces/public/share_saved_objects_to_space/components/selectable_spaces_control.tsx

```diff
@@ -46,22 +67,27 @@ export const SelectableSpacesControl = (props: Props) => {
   const { services } = useKibana();
   const { application, docLinks } = services;
 
+  const activeSpaceId = spaces.find((space) => space.isActiveSpace)!.id;
   const isGlobalControlChecked = selectedSpaceIds.includes(ALL_SPACES_ID);
   const options = spaces
     .sort((a, b) => (a.isActiveSpace ? -1 : b.isActiveSpace ? 1 : 0))
-    .map<SpaceOption>((space) => ({
-      label: space.name,
-      prepend: <SpaceAvatar space={space} size={'s'} />,
-      checked: selectedSpaceIds.includes(space.id) ? 'on' : undefined,
-      ['data-space-id']: space.id,
-      ['data-test-subj']: `sts-space-selector-row-${space.id}`,
-      ...(space.isActiveSpace ? activeSpaceProps : {}),
-      ...(isGlobalControlChecked && { disabled: true }),
-    }));
+    .map<SpaceOption>((space) => {
+      const checked = selectedSpaceIds.includes(space.id);
+      return {
+        label: space.name,
+        prepend: <SpaceAvatar space={space} size={'s'} />,
+        checked: checked ? 'on' : undefined,
+        ['data-space-id']: space.id,
+        ['data-test-subj']: `sts-space-selector-row-${space.id}`,
+        ...(isGlobalControlChecked && { disabled: true }),
+        ...(space.isPartiallyAuthorized && partiallyAuthorizedSpaceProps(checked)),
```

Comment:
I think we're missing tests for these "partially authorized" and "active space" scenarios. I think it'd be worthwhile to add explicit test cases for these scenarios since `activeSpaceProps` implicitly overrides `partiallyAuthorizedSpaceProps` props.

Reply:
Done in 9343ca6d0